### PR TITLE
Added support for @org/packages

### PR DESCRIPTION
**Pull request by thejameskyle**
Moving from #22

### CODE REVIEW

**Comment by thejameskyle** [comment]
This basically just adds support for the folder name being different than the package name.

File: lib/utils/checkUpdatedPackages.js [hunk 1]
```diff
@@ -28,22 +25,22 @@ module.exports = function checkUpdatedPackages(packagesLoc) {
...
     }
 
     // check if package has changed since last release
-    var diff = FORCE_VERSION.indexOf("*") >= 0 || FORCE_VERSION.indexOf(name) >= 0 ||
-               execSync("git diff " + lastTag + " -- " + getPackageLocation(packagesLoc, name));
+    var diff = FORCE_VERSION.indexOf("*") >= 0 || FORCE_VERSION.indexOf(pkg.name) >= 0 ||
+               execSync("git diff " + lastTag + " -- " + getPackageLocation(packagesLoc, pkg.name));
     if (diff) {
-      changedPackages.push(name);
+      changedPackages.push(pkg);
```
Comment (hzoo) [inline lib/utils/checkUpdatedPackages.js]: this should be `pkg.name` right

File: lib/commands/publish/updateChangedPackages.js [hunk 2]
```diff
@@ -16,19 +16,18 @@ function updateDepsObject(deps, changedPackages, version) {
...
 
-  changedPackages.forEach(function (name) {
-    var pkgLoc = getPackageLocation(packagesLoc, name) + "/package.json";
-    var pkg = require(pkgLoc);
+  changedPackages.forEach(function (pkg) {
+    var pkgLoc = getPackageLocation(packagesLoc, pkg.folder) + "/package.json";
 
     // set new version
-    pkg.version = version;
+    pkg.pkg.version = version;
```
Comment (hzoo) [inline lib/commands/publish/updateChangedPackages.js]: lol pkg.pkg looks weird

File: lib/utils/checkUpdatedPackages.js (hunk 1 above)
Comment (thejameskyle) [inline lib/utils/checkUpdatedPackages.js]: Nope, I need the whole `pkg` object elsewhere.

File: lib/commands/publish/updateChangedPackages.js (hunk 2 above)
Comment (thejameskyle) [inline lib/commands/publish/updateChangedPackages.js]: I didn't know what else to do :shrug:

**Comment by hzoo** [comment]
:+1: as much as I can check manually

**Comment by dendril** [comment]
When will you tag lerna to test the changes?

**Comment by thejameskyle** [comment]
I have a few more things to do before then, should be done within the next day or two

**Comment by dendril** [comment]
😱🎉 yeah!

**Comment by dchambers** [comment]
@thejameskyle, any update on being able to use this feature?

**Pull request merged**